charts/flyte/values.yaml

```diff
@@ -62,6 +63,50 @@ flyteadmin:
   additionalVolumes: {}
   additionalVolumeMounts: {}
 
+#
+# FLYTESCHEDULER SETTINGS
+#
+
+flytescheduler:
```

Comment:
I do not like that there are two nobs to control the scheduler... I've to enable workflow_scheduler AND flytescheduler to get flytescheduler to work... I think that's confusing...

What's the purpose of this enabled flag?
Shouldn't it be enough to say:
if workflow_scheduler.enabled and workflow_scheduler.type == "native"... then use this... or something like that?

Reply:
+1